server/plugin/command.go

```diff
@@ -43,6 +43,14 @@ const (
 	deleteAll = "delete-all"
 )
 
+type Features struct {
```

Comment:
@hanzei  not sure , but tried hope this is what you were referring to in this comment

Reply:
I think he's suggesting you alias it as an actual string like so:

```suggestion
type Features string
```